src/validation.cpp

```diff
@@ -1053,7 +1054,7 @@ CoinsViews::CoinsViews(
                             GetDataDir() / ldb_name, cache_size_bytes, in_memory, should_wipe)),
                         m_catcherview(MakeUnique<CCoinsViewErrorCatcher>(m_dbview.get())) {}
 
-void CoinsViews::InitCache()
+void CoinsViews::InitCache() EXCLUSIVE_LOCKS_REQUIRED(::cs_main)
```

Comment:
in commit cdf6afbbf1

Please don't add lock annotations in the cpp file. They will overwrite the ones in the header file

Reply:
Oops, stupid mistake.